templates/base-back.html.twig

```diff
@@ -2,6 +2,7 @@
 <html lang="fr">
     <head>
         <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Pour le front, il y a l'attribut `shrink-to-fit`, `<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`, qui n'a pas été conseillé par l'auditeur. J'ai essayé de le mettre pour voir et je n'ai pas vu de différence. J'ai l'impression que c'est spécifique à Safari. Faut-il le mettre ?

Reply:
D'après ce que je vois ici : https://stackoverflow.com/questions/33767533/what-does-the-shrink-to-fit-viewport-meta-attribute-do
Sur la deuxième réponse, ils encouragent plutôt à l'enlever.
Peut-être plutôt l'enlever de `base.html.twig` ? :)

Reply:
ok c'est fait